main/src/cgeo/geocaching/utils/DisplayUtils.java

```diff
@@ -9,6 +9,8 @@
 
 public class DisplayUtils {
 
+    private static final float THIN_LINE = 5f;
```

Comment:
Looking at the code changes above, I would expect the constant to be `7f`.

Reply:
Four different widths have been used in the past:
- 5f * density
- 8f * density
- 3
- 7
the first two being for the same, but in different map types, the latter two being absolute values instead of density related.

I merged this into `getThinLineWith()` and `getThinLineWidth() / 2` (although agreeable that I could invent two different methods for this - will update the PR accordingly).